src/login/AccountActivationMessage.jsx

```diff
@@ -59,7 +60,7 @@ const AccountActivationMessage = (props) => {
   return activationMessage ? (
     <Alert
       id="account-activation-message"
-      className="mb-4"
+      className="mb-5"
```

Comment:
Just curious why would we make a style change?

Reply:
It was inconsistent with the login form design. The gap between username/email field and banner was too low and different from the LoginFailure gap